modules/core/src/shaderlib/picking/picking.js

```diff
@@ -1,24 +1,24 @@
-import {picking, createModuleInjection} from '@luma.gl/core';
+import {picking} from '@luma.gl/core';
 
-createModuleInjection('picking', {
-  hook: 'vs:DECKGL_FILTER_COLOR',
-  injection: `
-    picking_setPickingColor(geometry.pickingColor);
-    // for picking depth values
-    picking_setPickingAttribute(geometry.position.z);
-`
-});
+export default Object.assign(
+  {
+    inject: {
+      'vs:DECKGL_FILTER_COLOR': `
+      picking_setPickingColor(geometry.pickingColor);
```

Comment:
Total nit: I usually format the injection code with the indentation it will need in the final glsl...

Reply:
Oh sure, I thought matched it. Is it 4 spaces?